compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2595,7 +2596,7 @@ impl<'a> Parser<'a> {
     }
 
     /// Parses the condition of a `if` or `while` expression.
-    fn parse_expr_cond(&mut self) -> PResult<'a, P<Expr>> {
+    fn parse_expr_cond(&mut self, edition: Edition) -> PResult<'a, P<Expr>> {
```

Comment:
Should document which span this edition parameter is attached to.

Reply:
Done